gstd/src/msg/async.rs

```diff
@@ -36,15 +36,17 @@ use futures::future::FusedFuture;
 /// To interrupt a program execution waiting for a reply on a previous message,
 /// one needs to call an `.await` expression.
 /// The initial message that requires a reply is sent instantly.
-/// Function `send_and_wait_for_reply` returns `CodecMessageFuture` which
+/// Function `send_for_reply` returns `CodecMessageFuture` which
 /// implements `Future` trait. Program interrupts until the reply is received.
 /// As soon as the reply is received, the function checks it's exit code and
 /// returns `Ok()` with decoded structure inside or `Err()` in case of exit code
 /// does not equal 0. For decode-related errors (<https://docs.rs/parity-scale-codec/2.3.1/parity_scale_codec/struct.Error.html>),
 /// Gear returns the native one after decode.
 pub struct CodecMessageFuture<T> {
-    waiting_reply_to: MessageId,
-    phantom: PhantomData<T>,
+    /// Waiting reply to this the message id
+    pub waiting_reply_to: MessageId,
+    /// Marker
+    pub phantom: PhantomData<T>,
```

Comment:
No one uses this marker.
```suggestion
    _phantom: PhantomData<T>,
```